### ISSUE TITLE
Run black on all existing code

### ISSUE THREAD

**Issue by gaffney2010** [Axelrod-Python/Axelrod#1312]
If we're using black to format, I'd prefer to run on existing code.  If we format as we go it can be harder to see why we're changing something.  It caused me some merge conflicts when I ran black, changing lines, which were unchanged in origin, then were changed.

**Comment by drvinceknight** [comment]
I'm in favour. 

We can add a check to the CI to check all files are in the required format (for example: https://github.com/drvinceknight/Nashpy/blob/master/.github/workflows/config.yml#L34).

We'd need to add something to the contribution docs as well.

Something to consider though is the relatively massive PR that @marcharper has on the go #1288 this would potentially make that really painful to sort out so it could be worth doing after that goes in.

**Comment by gaffney2010** [comment]
Sounds good.  I will work on after #1288 goes through.

**Comment by marcharper** [comment]
#1288 is going to have a lot of merge conflicts anyway, and it's waiting on another PR, so feel free to work on whatever regardless.
